docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -0,0 +1,196 @@
+to request physical storage. In this exercise, you create a PersistentVolumeClaim
+that requests a volume of at least three gibibytes that can provide read-write
+access for at least one Node.
+
+Here is the configuration file for the PersistentVolumeClaim:
+
+{% include code.html language="yaml" file="task-pv-claim.yaml" ghlink="/docs/tasks/configure-pod-container/task-pv-claim.yaml" %}
+
+Create the PersistentVolumeClaim:
+
+    kubectl create -f http://k8s.io/docs/tasks/configure-pod-container/task-pv-claim.yaml
+
+After you create the PersistentVolumeClaim, the Kubernetes control plane looks
+for a PersistentVolume that satisfies the claim's requirements. If the control
+plane finds a suitable PersistentVolume, it binds the claim to the volume.
+
+Look again at the PersistentVolume:
+
+    kubectl get pv task-pv-volume
+
+Now the output shows a `STATUS` of `Bound`.
+
+    kubectl get pv task-pv-volume
+    NAME             CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS    CLAIM                   REASON    AGE
+    task-pv-volume   10Gi       RWO           Retain          Bound     default/task-pv-claim             8m
+
+Look at the PersistentVolumeClaim:
+
+    kubectl get pvc task-pv-claim
+
+The output shows that the PersistentVolumeClaim is bound to your PersistentVolume,
+`task-pv-volume`.
+
+    NAME            STATUS    VOLUME           CAPACITY   ACCESSMODES   AGE
+    task-pv-claim   Bound     task-pv-volume   10Gi       RWO           5s
+
+## Creating a Pod
+
+The next step is to create a Pod that uses your PersistentVolumeClaim as a volume.
+
+Here is the configuration file for the Pod:
+
+{% include code.html language="yaml" file="task-pv-pod.yaml" ghlink="/docs/tasks/configure-pod-container/task-pv-pod.yaml" %}
+
+Notice that the Pod's configuration file specifies a PersistentVolumeClaim, but
+it does not specify a PersistentVolume. From the Pod's point of view, the claim
+is a volume.
+
+Create the Pod:
+
+    kubectl create -f http://k8s.io/docs/tasks/configure-pod-container/task-pv-pod.yaml
+
+Verify that the Container in the Pod is running;
+
+    kubectl get pod task-pv-pod
+
+Get a shell to the Container running in your Pod:
+
+    kubectl exec -it task-pv-pod -- /bin/bash
+
+In your shell, verify that nginx is serving the `index.html` file from the
+hostPath volume:
+
+    root@task-pv-pod:/# apt-get update
+    root@task-pv-pod:/# apt-get install curl
+    root@task-pv-pod:/# curl localhost
+
+The output shows the text that you wrote to the `index.html` file on the
+hostPath volume:
+
+    Hello from Kubernetes storage
+
+{% endcapture %}
+
+
+{% capture discussion %}
+
+## Access control
+
+Storage configured with a group ID (GID) allows writing only by Pods using the same
+GID. Mismatched or missing GIDs cause permission denied errors. To reduce the
+need for coordination with users, an administrator can annotate a PersistentVolume
+with a GID. Then the GID is automatically added to any Pod that uses the
```

Comment:
What you said is clear in the text, but something about it was surprising to me when I read it.  I think because in the rest of the document we say the Pod uses the PersistentVolumeClaim, but don't say it uses the Volume.  This is fine.  An alternative for your consideration:

>
added to any Pod that has claimed the PersistentVolume